app/components/search/SearchResults/SearchResults.tsx

```diff
@@ -39,6 +39,19 @@ const SearchResults = ({
     };
   }, [googleMapObject, centerCoords]);
 
+  // Scroll to the top and move focus to the first result after pagination
+  const firstResultRef = useRef<HTMLDivElement | null>(null);
+  useEffect(() => {
+    window.scrollTo({
+      top: 0,
+      behavior: "smooth",
+    });
```

Comment:
Do we need a ref since the first result is always in the same scroll position? Could we set `top` here to the absolute position in the document? I can't quite remember if this positioning is absolute or relative.

Reply:
Oh we don't! Now that the focus has shifted to the top element, the user is automatically taken to the top result without the scroll behavior.